docker-compose-e2e.yaml

```diff
@@ -0,0 +1,426 @@
+version: "3.9"
+services:
+  # Keycloak
+  zac-keycloak:
+    image: quay.io/keycloak/keycloak:22.0.1
+    container_name: zac-keycloak-e2e
```

Comment:
Why (cc @Jorann ) these container names? Why can we not use our normal docker-compose.yml with the container names like `zac-keycloak`?

It's not like we already have any of these Docker containers running in GitHub right?

Reply:
Then you would not be able to run it locally, because just specifiying the `--project_name` does not seem to create a separate volume. (you get an error that the volume is already in use by another container)

We chose to be able to run it locally as well.

Reply:
We can also remove all `container_name` attributes from our (one) `docker-compose.yaml` file. Looks like I have to do this anyway because the TestContainers framework does not allow this when you want to use a Docker Compose file..